web/client/plugins/manager/ManagerMenu.jsx

```diff
@@ -108,6 +121,7 @@ class ManagerMenu extends React.Component {
 module.exports = {
     ManagerMenuPlugin: assign(connect((state) => ({
         enableRulesManager: isRulesManagerConfigured(state),
+        enableImporter: () => isImporterConfigured(state),
```

Comment:
why not simply isImporterConfigured(state) ?

Reply:
Because I did an error, good catch :+1:

Reply:
This should be solved